### PR TITLE
Dvd updates

### PR DESCRIPTION
**Pull request by mustafa-x**
Updating links, and new spotify link.

### CODE REVIEW

File: src/content/en/shows/designer-vs-developer/podcast/DVDS1E01-creating-a-collaborative-environment.md [hunk 1]
```diff
@@ -33,13 +33,17 @@ In this episode:
 * Designers lack of version control
 
 <audio style="width: 100%" 
-src="https://storage.googleapis.com/http-203-podcast/designer-vs-developer/dvd-s01-e01-creative-collaborations.mp3" 
+src="https://storage.googleapis.com/http-203-podcast/designer-vs-developer/
+dvd-s01-e01-creative-collaborations.mp3"
```
Comment (petele) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS1E01-creating-a-collaborative-environment.md]: New line here will break the link, this needs to stay on one line.

File: src/content/en/shows/designer-vs-developer/podcast/DVDS1E02-balancing-creativity-with-user-testing.md [hunk 2]
```diff
@@ -35,13 +35,17 @@ In this episode:
 * How we can use sandwiches as a source of inspiration
 
 <audio style="width: 100%" 
-src="https://storage.googleapis.com/designer-vs-developer-podcast/DVD-S01-E02-balancing-creativity-with-user-testing.mp3" 
+src="https://storage.googleapis.com/designer-vs-developer-podcast/
+DVD-S01-E02-balancing-creativity-with-user-testing.mp3"
```
Comment (petele) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS1E02-balancing-creativity-with-user-testing.md]: New line here will break the link, this needs to stay on one line.

File: src/content/en/shows/designer-vs-developer/podcast/DVDS1E03-Adopting-the-native-language-of-the-web.md [hunk 3]
```diff
@@ -34,14 +34,19 @@ In this episode:
 * How the web can retain its soul.
 
 <audio style="width: 100%" 
-src="https://storage.googleapis.com/designer-vs-developer-podcast/DVDS1E03-Adopting-the-native-language-of-the-web.mp3" controls preload="none">
+src="https://storage.googleapis.com/designer-vs-developer-podcast/
+DVDS1E03-Adopting-the-native-language-of-the-web.mp3" controls preload="none">
```
Comment (petele) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS1E03-Adopting-the-native-language-of-the-web.md]: New line here will break the link, this needs to stay on one line.

File: src/content/en/shows/designer-vs-developer/podcast/DVDS1E04-are-designers-born-or-made.md [hunk 4]
```diff
@@ -33,12 +33,18 @@ In this episode:
 * What is it like as a developer to have a design eye?
 
 <audio style="width: 100%" 
-src="https://storage.googleapis.com/designer-vs-developer-podcast/DVDS1E04-Are-designers-and-developers-born-or-made.mp3" controls preload="none">
+src="https://storage.googleapis.com/designer-vs-developer-podcast/
+DVDS1E04-Are-designers-and-developers-born-or-made.mp3" controls
```
Comment (petele) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS1E04-are-designers-born-or-made.md]: New line here will break the link, this needs to stay on one line.

File: src/content/en/shows/designer-vs-developer/podcast/DVDS1E05-who-is-responsible-for-design.md [hunk 5]
```diff
@@ -34,11 +34,13 @@ In this episode:
 * Material design system
 
 <audio style="width: 100%" 
-src="https://storage.googleapis.com/designer-vs-developer-podcast/DVDS1E05-who-is-responsible-for-design.mp3" 
+src="https://storage.googleapis.com/designer-vs-developer-podcast/
+DVDS1E05-who-is-responsible-for-design.mp3"
```
Comment (petele) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS1E05-who-is-responsible-for-design.md]: New line here will break the link, this needs to stay on one line.

File: src/content/en/shows/designer-vs-developer/podcast/DVDS1E06-are-design-tools-fit-for-purpose.md [hunk 6]
```diff
@@ -37,13 +37,17 @@ href="https://github.com/GoogleChrome/howto-components">HowTo: Components</a>’
 * Exploring the best practice and patterns of the web.
 
 <audio style="width: 100%"
-src="https://storage.googleapis.com/designer-vs-developer-podcast/DVDS1E06-are-design-tools-fit-for-purpose.mp3" controls preload="none">
+src="https://storage.googleapis.com/designer-vs-developer-podcast/
+DVDS1E06-are-design-tools-fit-for-purpose.mp3" controls preload="none">
```
Comment (petele) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS1E06-are-design-tools-fit-for-purpose.md]: New line here will break the link, this needs to stay on one line.

**Comment by mustafa-x** [comment]
fixed those changes and also added the new episode.

File: src/content/en/shows/designer-vs-developer/podcast/DVDS3E02-designing-for-standards.md [hunk 7]
```diff
@@ -1,13 +1,13 @@
 project_path: /web/_project.yaml
 book_path: /web/shows/_book.yaml
-description: Bruce Lawson talks about designing for standards, and not just one browser. We also cover Styleable, a CSS preprocessor that he is working on at Wix.
+description: Bruce Lawson talks about designing for standards, and not just one browser. We also cover Styleable, a CSS preprocessor that they are working on at Wix.
```
Comment (samdutton) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS3E02-designing-for-standards.md]:
```suggestion
description: Bruce Lawson talks about designing for standards, and not just one browser. We also cover Stylable, a CSS preprocessor developed at Wix.
```

https://stylable.io

File: src/content/en/shows/designer-vs-developer/podcast/DVDS3E06-how-to-design-a-vr-game.md [hunk 8]
```diff
@@ -20,7 +20,7 @@ the two, providing take aways, solutions to workflows, tools and
 discussions on everyday struggles.
 
 Mustafa speaks to Ada Rose Cannon, Developer Advocate for the
-Samsung Internet Web Browser, about her passion about Virtual
+Samsung Internet Web Browser, about their passion about Virtual
```
Comment (samdutton) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS3E06-how-to-design-a-vr-game.md]:
```suggestion
Samsung Internet Web Browser, about their passion for Virtual
```

File: src/content/en/shows/designer-vs-developer/podcast/DVDS4E06-pwas-firebase.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+<img class="attempt-right"
+  src="/web/shows/designer-vs-developer/podcast/images/dvd-s04-e06-cover.jpg">
+  
+In this episode:
+
+* PWA's
+* Designing for speed
+* Firebase and its size
+* [Progressive WordPress (PWP)](http://bit.ly/2Jl2nMK) 
+* [Cloud firestore](http://bit.ly/2AMVu1u)
```
Comment (samdutton) [inline src/content/en/shows/designer-vs-developer/podcast/DVDS4E06-pwas-firebase.md]:
```suggestion
* [Cloud Firestore](http://bit.ly/2AMVu1u)
```

**Pull request merged**